app/src/main/java/org/oppia/android/app/player/state/SelectionInteractionView.kt

```diff
@@ -92,8 +94,7 @@ class SelectionInteractionView @JvmOverloads constructor(
           )
           .build()
       SelectionItemInputType.RADIO_BUTTONS ->
-        BindableAdapter.SingleTypeBuilder
-          .newBuilder<SelectionInteractionContentViewModel>()
+        singleTypeBuilderFactory.create<SelectionInteractionContentViewModel>()
           .registerViewBinder(
```

Comment:
I believe it will also crash here with the same error.

Reply:
The same solution as mentioned above will be needed, I think.